start-site/src/main/resources/application.yml

```diff
@@ -161,15 +161,15 @@ initializr:
         versionProperty: vaadin.version
         mappings:
           - compatibilityRange: "[2.1.0.RELEASE,2.6.0-M1)"
-            version: 14.10.2
+            version: 14.10.3
           - compatibilityRange: "[2.6.0-M1,2.7.0-M1)"
-            version: 23.2.15
+            version: 23.2.14
```

Comment:
This looks like a downgrade. Should it be `.16`?

Reply:
oops, that is a mistake.. actually 23.2 is not supported any more. i should not touch that version.